cmd/provider-services/cmd/status.go

```
@@ -27,23 +41,33 @@ func statusCmd() *cobra.Command {
+	cctx, err := cli.GetClientTxContext(cmd)
 	if err != nil {
 		return err
 	}
 
-	gclient, err := apclient.NewClient(ctx, cl, addr)
+	var queryClient v1beta3.QueryClient
+	if cl != nil {
+		queryClient = cl.Query()
+	}
+
+	paddr, err := providerFromFlags(cmd.Flags())
```

golangci-lint (2.5.0)

[error] ineffectual assignment to err

(ineffassign)